jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -179,6 +198,26 @@ teardown() {
     [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:postgres://host:3306/postgres?ssl=true\&amp\;sslmode=allow" ]
 }
 
+@test "verify if EJB_TIMER is correctly configured with data-sources using URL with multiple params when driver is postgres with RHPAM_XA_CONNECTION_PROPERTY_Url" {
+    local expected_timer_service="EJB_TIMER"
+    local expected_datasources="EJB_TIMER,RHPAM"
+    export DATASOURCES="RHPAM"
+    export RHPAM_DRIVER="postgresql"
+    export RHPAM_NONXA="false"
+    export RHPAM_XA_CONNECTION_PROPERTY_Url="jdbc:postgres://host:3306/postgres?ssl=true&amp;sslmode=allow"
+    configure_EJB_Timer_datasource >&2
+    echo "EJBTimer url is ${EJB_TIMER_XA_CONNECTION_PROPERTY_Url} " >&2
+    echo "RHPAM_XA_CONNECTION_PROPERTY_Url is ${RHPAM_XA_CONNECTION_PROPERTY_Url} " >&2
+    [ "${TIMER_SERVICE_DATA_STORE^^}" = "${expected_timer_service}" ]
+    [ "${DATASOURCES}" = "${expected_datasources}" ]
+    [ "${EJB_TIMER_DRIVER}" = "${RHPAM_DRIVER}" ]
+    # we do not expect that this var is set anymore, since we're using URL property directly
+    [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_PinGlobalTxToPhysicalConnection}" = "" ]
+    [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_EnabledSslProtocolSuites}" = "" ]
+    # the URL property must be set
+    [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:postgres://host:3306/postgres?ssl=true\&amp\;sslmode=allow" ]
```

Comment:
Missing the check for the RHPAM xa Url

Reply:
Check on RHPAM_URL added

Reply:
in this case, it is missing to check that RHPAM_URL is empty, it is a XA datasource, RHPAM_URL is for nonXA